shaders/commonProfileShaders.cpp

```diff
@@ -55,11 +56,17 @@ namespace GLTF
         ((std::istream*)png_get_io_ptr(pngPtr))->read((char*)data, length);
     }
 #endif
+	static std::unordered_map<std::string, bool> haveAlphas;		// Caching to remove up to 98% execution time
```

Comment:
The upshot is it's a static variable, so the destructor shouldn't be called until the program exits, and if you're in debug mode, I'm assuming you can just kill it.